package.json

```diff
@@ -24,13 +24,13 @@
         "timeago.js": "^4.0.2"
     },
     "peerDependencies": {
-        "@emotion/core": "^10.0.27",
-        "emotion": "^10.0.27",
+        "@emotion/core": "^10.0.28",
         "@guardian/src-button": "^0.17.0",
         "@guardian/src-foundations": "^0.17.0",
         "@guardian/src-link": "^0.17.0",
         "@guardian/src-svgs": "^0.17.0",
         "@guardian/src-text-input": "^0.17.0",
+        "emotion": "^10.0.27",
```

Comment:
Is there an `emotion@10.0.28`? Does it matter?

Reply:
Nope, there isn't, and I don't think it matters, they look like they're partly aligned but versioned independently https://github.com/emotion-js/emotion/blob/master/packages/core/CHANGELOG.md#patch-changes